[2023-09-04T06:01:11.479+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [queued]>
[2023-09-04T06:01:11.497+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [queued]>
[2023-09-04T06:01:11.497+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:01:11.521+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2021-12-30 00:00:00+00:00
[2023-09-04T06:01:11.531+0000] {standard_task_runner.py:57} INFO - Started process 4676 to run task
[2023-09-04T06:01:11.537+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2021-12-30T00:00:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpt05n0enz']
[2023-09-04T06:01:11.545+0000] {standard_task_runner.py:85} INFO - Job 161: Subtask delete_data_from_table
[2023-09-04T06:01:11.661+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:01:11.835+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2021-12-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-30T00:00:00+00:00'
[2023-09-04T06:01:11.839+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2021-12-30' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:01:11.860+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:11.877+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:11.903+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2021-12-30' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:01:11.912+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:01:11.935+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20211230T000000, start_date=20230904T060111, end_date=20230904T060111
[2023-09-04T06:01:12.008+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:01:12.167+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:00.746+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [queued]>
[2023-09-04T06:04:00.788+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [queued]>
[2023-09-04T06:04:00.789+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:00.926+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2021-12-30 00:00:00+00:00
[2023-09-04T06:04:00.942+0000] {standard_task_runner.py:57} INFO - Started process 5993 to run task
[2023-09-04T06:04:00.960+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2021-12-30T00:00:00+00:00', '--job-id', '706', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp1wmr2qgg']
[2023-09-04T06:04:00.969+0000] {standard_task_runner.py:85} INFO - Job 706: Subtask delete_data_from_table
[2023-09-04T06:04:01.173+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2021-12-30T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:01.385+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2021-12-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-30T00:00:00+00:00'
[2023-09-04T06:04:01.388+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2021-12-30' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:01.414+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:01.438+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:01.465+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2021-12-30' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:01.475+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:01.503+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20211230T000000, start_date=20230904T060400, end_date=20230904T060401
[2023-09-04T06:04:01.583+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:01.636+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
